apps/meteor/app/file-upload/server/methods/sendFileMessage.ts

```diff
@@ -9,14 +9,25 @@ import { canAccessRoom } from '../../../authorization/server/functions/canAccess
 import { SystemLogger } from '../../../../server/lib/logger/system';
 import { omit } from '../../../../lib/utils/omit';
 
+const validateFileRequiredFields = (file: Partial<IUpload>): void => {
```

Comment:
```suggestion
function validateFileRequiredFields(file: Partial<IUpload>): asserts file is AtLeast<IUpload, '_id' | 'name' | 'type' | 'size'> {
```
You won't need as many typecasts with this.
`AtLeast` can be imported from core-typings.